python/tvm/relay/backend/vm.py

```diff
@@ -150,9 +150,7 @@ def lower(self, mod, target=None, target_host=None):
             target, target_host, target_is_dict_key=False
         )
 
-        tophub_context = self._tophub_context(target)
```

Comment:
Well TIL this exists...

Reply:
Whoops, accidentally left this in. See https://github.com/apache/tvm-rfcs/pull/59 for some context.